.run/start jenkins.run.xml

```diff
@@ -1,48 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
Since the steps of the guide make us create a new Run configuration when we open on IntelliJ, why would we save this particular one ? Unless it is necessary to have it for some remote testing or to run the project on the cli, I don't think it does anything

Reply:
when you open intellij you should get this working run configuration and not have to do anything.

i.e. you should be able to start developing immediately without having to follow a guide

Reply:
That wasn't the case for me. Here are the steps I took:
- Cloned the project from my fork
- Opened it on IntelliJ
- Ran the project with the existing `start jenkins.run.xml` run configuration. Outputs a stack of exceptions and doesn't run.
- The `docs/CONTRIBUTING.md` suggests that we have to create a new run configuration. which I did but it still didn't run.
- I had to modify the Java Options properties and change the value of `jenkins.version` to the one I had installed.
- After that the project ran properly.
- The guide tells us to open <http://localhost:8080/jenkins/configuration-as-code> to test the plugin locally
- Since I had Jenkins already running on port 8080 I had to map it to port 8085

I hope this helps clarify the difficulties I had running the project and the reason for this PR.